### ISSUE TITLE
Impossible to login after migration 

### ISSUE THREAD

**Issue by nicfab** [BookStackApp/BookStack#5314]
### Attempted Debugging

- [X] I have read the debugging page

### Searched GitHub Issues

- [X] I have searched GitHub for the issue.

### Describe the Scenario

I installed (manual) Bookstack from scratch on a new server.
Then, I restored the database.
Accessing from the web, after inputting login, password, and 2FA, I see "_An Error Occurred. An unknown error occurred", as for the image below.

![sshot_2024-11-10_13 35 59](https://github.com/user-attachments/assets/a28da64d-364c-4aa1-a7a4-b430507c8d5c)

Can you help me?

### Exact BookStack Version

24.10.1

### Hosting Environment

PHP8.2

**Comment by ssddanbrown** [comment]
Hi @nicfab,
Please check your error log for details regarding the error: https://www.bookstackapp.com/docs/admin/debugging/#error-log-file

If I had to guess without error detail, it's maybe due to the new instance/setup not using the same `APP_KEY` value, which is used for MFA encryption. The `APP_KEY` should be the same as your old instance.
This is only a guess though, check the log for actual information.

**Comment by nicfab** [comment]
Hi @ssddanbrown!
Thank you very much for your quick reply.
I set the `APP_KEY` of the old instance, and I can log in now, although I should set again my name instead of Admin, my photo, and something else.

**Comment by nicfab** [comment]
@ssddanbrown 
I need your help again.
I am still migrating my instance.
On my new server, I installed MariaDB (mariadb  Ver 15.1 Distrib 10.11.6-MariaDB, for debian-linux-gnu (x86_64) using  EditLine wrapper).
After the database backup, I restored it; indeed, accessing maridab I see the tables.
However, when running `php artisan migrate` under the `bookstack` folder, I receive some errors, and it seems that there is no access to the database.
Below the screenshot

![sshot_2024-11-13_14 25 21](https://github.com/user-attachments/assets/df224d9e-6c2f-44d3-9663-444a8ac199cd)

What should I do?
